pkg/setting/setting_oauth.go

```diff
@@ -1,5 +1,10 @@
 package setting
 
+type OAuthOrgInfo struct {
```

Comment:
What does this do? I don't see it used anywhere.

Reply:
Good catch.  That is not needed anymore.